#request
string inpString
---
#response
string outString
